extensions/amp-sidebar/1.0/amp-sidebar.md

```diff
@@ -0,0 +1,348 @@
+---
+$category@: layout
+formats:
+  - websites
+  - stories
+  - email
+  - stories
+teaser:
+  text: Provides a way to display meta content intended for temporary access such as navigation, links, buttons, menus.
+---
+<!---
+Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# <a name="amp-sidebar"></a> `amp-sidebar`
```

Comment:
Can you add a disclaimer here stating that this version is still super early stage and should not be used. If you start the paragraph with:

```Important: ...```

It'll render as a callout.

Reply:
@sebastianbenz What do you think about replacing "should not be used" with "should be used with caution"? This is more in line with the [experimental documentation](https://amp.dev/documentation/guides-and-tutorials/learn/experimental/), and gives the opportunity to find bugs earlier on.